techrecord.md

```diff
@@ -77,3 +77,55 @@ All temporal objects should be information stable. All methods that increase or
 
 Negative durations make no sense. Therefore there is a need to have both a `plus` and a `minus` method for date/time arithmetics.
 Only integer units are supported in durations and arithmetic. Expressing fractions requires the use of smaller units.
+
+### The properties on Instant should be getXXX() methods
+
+This comes out of [#139](https://github.com/tc39/proposal-temporal/issues/139). It's actually true that the epochNanoseconds and their ilk aren't really the value of the Instant and that the value of the Instant shouldn't be exposed. There is however a clear relation between an Instant and the epoch, so that `fromEpochNanoseconds()` and `getEpochNanoseconds()` should be the way forward.
+
+### TimeZone objects should be a thing
+
+There have been multiple requests that there is an interface to the time-zone data directly. For that reason we have introduced TimeZone objects. They stringify to the IANA-name of the timezone or the Offset-String in case of a pure offset.
+
+### OffsetDateTime
+
+Since `TimeZone` objects are now a thing and have all the information necessary to distinguish between pure offsets and actual IANA time-zones the distinction between `OffsetDateTime` and `ZonedDateTime` has become meaningless. Therefore we are dropping `OffsetDateTime`.
```

Comment:
What about the other major difference between OffsetDateTime and ZoneDateTime—their serialization? The latter writes out and reads in a bracketed time zone like "[America/New_York]", but the former does not.

Reply:
A tz object knows what “kind” it is while a string doesn’t. So by having tz objects that problem goes away.

Reply:
It sounds like you want `ZonedDateTime.prototype.toString` to output a different _format_ based on whether or not its zone is a fixed offset (i.e., with or without a bracketed name), but I thought that approach had already been rejected.

And this change also introduces a gap—there is no longer a convenient means to identify the offset of a ZonedDateTime instance. The best I can come up with is `zoned.toString().match(/[+-]\d\d:\d\d/)?.[0]`, and that's obviously not something we should recommend.

Reply:
No because the timezone property references an object now and that can give you the offset (plus all possible offsets in a year and a bunch more)

Reply:
But that object is a TimeZone independent of the ZonedDateTime from which it was retrieved and thus ignorant of the instant at which the UTC offset is desired... you'd have to pull out both the time zone and the instant like `zoned.getTimeZone().offset(zoned.getInstant())`, which is arguably worse than string parsing (and certainly more complex).